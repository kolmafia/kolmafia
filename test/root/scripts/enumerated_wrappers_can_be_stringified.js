const { printHtml } = require("kolmafia");

printHtml(JSON.stringify(Item.get("lime")));
printHtml(JSON.stringify(Skill.get("Ode to Booze")));

var test = JSON.parse(JSON.stringify(Monster.get("zmobie")));
printHtml(`Flip flop: ${test.image === Monster.get("zmobie").image}`);