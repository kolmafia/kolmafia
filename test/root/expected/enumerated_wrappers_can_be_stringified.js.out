{"id":333,"name":"lime","plural":"limes","descid":"839503923","image":"lime.gif","smallimage":"lime.gif","levelreq":2,"quality":"decent","adventures":"2","muscle":"0","mysticality":"0","moxie":"0","fullness":1,"inebriety":0,"spleen":0,"minhp":0,"maxhp":0,"minmp":0,"maxmp":0,"dailyusesleft":15,"notes":"","quest":false,"gift":false,"tradeable":true,"discardable":true,"combat":false,"combatReusable":false,"usable":false,"reusable":false,"multi":false,"fancy":false,"pasteable":true,"smithable":false,"cookable":true,"mixable":true,"candy":false,"candyType":"none","chocolate":false,"potion":false,"seller":"none","buyer":"none","nameLength":4,"noobSkill":"Small Intestine","tcrsName":"lime","skill":"none","recipe":"none"}
{"id":6014,"name":"The Ode to Booze","type":"buff","level":12,"image":"odetobooze.gif","traincost":7500,"class":"Accordion Thief","libram":false,"passive":false,"buff":true,"combat":false,"song":false,"expression":false,"walk":false,"summon":false,"permable":true,"dailylimit":-1,"dailylimitpref":"","timescast":0}
Flip flop: true